src/main/java/com/akademia/educators_back/entity/UserEntity.java

```diff
@@ -0,0 +1,23 @@
+package com.akademia.educators_back.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "users")
+public class UserEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private List<Answer> answers;
```

Comment:
Nie wrzucamy kodu, który się nie uruchamia :) jeśli nie ma jeszcze Encji to możemy jej nie dodawać i najlepiej dodać //TODO z komentarzem